src/components/TheHeader.vue

```diff
@@ -3,7 +3,7 @@
     <div
       :class="[
         'header__container',
-        { 'header__container--open': isNavigationOpen },
+        { 'header__container--open': isNavigationDrawerOpen },
```

Comment:
```suggestion
        { 'header__container--open': isNavigationOpen },
```

I think we can stick to original naming, or is there some consideration for renaming?

Reply:
I used NavigationDrawer just because I find it more specific and precise. Navigation is ambiguous since it may refer to bottom navigation or top navigation. Moreover, navigation on desktop is visible all the time, only the drawer is something that can be opened or closed. If this doesn't convince you I'll go back to the previous name

Reply:
I don't see a need for renaming as nothing has fundamentally changed, but if you insist on something more precise, you can use Menu instead of Drawer. Just wanted to avoid introducing new names to codebase

Reply:
Great suggestion, I renamed it to isMobileMenuOpen